_storage/digital-ocean-deploy.md

```diff
@@ -73,8 +73,39 @@ In this task you create a **Space** which is where Gaia stores your files.
 
    After a moment, your Space is up and running.
 
+6. Enable **File Listing** for your space
 
-## Task 2: Create a DigitalOcean droplet
+   Click on your space, then click on the settings tab and enable File Listing.
+
+## Task 2: Grant Read Permission with a Bucket Policy
+
+1. Install and configure **s3cmd** on your development machine by following these instructions:
+
+   https://www.digitalocean.com/docs/spaces/resources/s3cmd/
```

Comment:
I'd like to talk to you about this, the instructions for s3cmd are pretty bad. They send the user to the API page but are unclear what to use there. Neither my personal access token or the space key worked.

Reply:
Sorted it out...they want you to omit the actual name of the space and just use the sub portion of the URL. I got it to work.